gpu-miner.c

```diff
@@ -252,3 +232,74 @@ int main() {
  
 	return 0;
 }
+
+// Perform global_item_size * iter_per_thread hashes
+// Return -1 if a block is found
+// Else return the hashrate in MH/s
+double grindNonces(size_t global_item_size, size_t iter_per_thread) {
```

Comment:
typically the helper functions are put at the top of the file, at least in Nebulous projects anyway.